web/themes/joinup/prototype/sass/components/_featured.sass

```diff
@@ -141,6 +141,28 @@
   margin: 10px
   z-index: 99
 
+// Style social media links for user
+.featured__social-media
```

Comment:
There is already social media component in _footer-social-media.sass, but the styling is completely different, so I decided to create a new component - inline-social-media